tests/test_strip_html.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from LSP.plugin.core.strip_html import strip_html
+
+
+class StripHtmlTest(unittest.TestCase):
+    def test_strip_html(self):
+        result = strip_html("<p>Hello <b>world</b><span id='hello'>!</span></p>")
+        expected = "Hello world!"
```

Comment:
Does it handle <br>'s or just block elements (should create new lines)?

I'm honestly surprised you went this way. It's a very heavy approach and not sure if really reliable. I was thinking that we could just show the original markdown markup which should be pretty readable itself.

Reply:
++ agree, will just show the original tooltip text without parsing

Reply:
https://github.com/sublimelsp/LSP/pull/2018/commits/3779a1345fc2ec8a511608e9daedd6ae15bc20de